gammapy/cube/basic_cube.py

```diff
@@ -0,0 +1,74 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions to perform basic functions for map and cube analysis.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ..maps import WcsNDMap
+
+
+def fill_map_counts(event_list, ndmap):
+    """Fill a ``WcsMap` with events from an EventList.
+
+     The energy of the events is used for a non-spatial axis homogeneous to energy.
+     The other non-spatial axis names should have an entry in the colum names of the ``EventList``
+
+     Parameters
+     ----------
+     event_list : `~gammapy.data.EventList`
+             the input event list
+     ndmap : `~gammapy.maps.Map`
```

Comment:
Here you use `ndmap`, but I think the code should also work for the sparse maps, i.e. we don't want "nd" in the argument name here. On the other hand, just using "map" has the disadvantage that we're clobbering the Python built-in "map" function, which is not recommended, by default flagged by Python linters.
We will have that issue everywhere. I'm not sure what is best here. @woodmd @registerrier ?